plugins/resource_monitor_plugin/include/eosio/resource_monitor_plugin/file_space_handler.hpp

```diff
@@ -104,18 +119,24 @@ namespace eosio::resource_monitor {
             ("ec", ec.value())
             ("message", ec.message()));
 
-         auto shutdown_available = (100 - shutdown_threshold) * (info.capacity / 100); // (100 - shutdown_threshold)/100 is the percentage of minimum number of available bytes the file system must maintain  
-         auto warning_available = (100 - warning_threshold) * (info.capacity / 100);
+         uintmax_t shutdown_available = shutdown_absolute;
+         uintmax_t warning_available = warning_absolute;
+         if (shutdown_absolute == 0) {
+            shutdown_available = (100 - shutdown_threshold) * (info.capacity / 100); // (100 - shutdown_threshold)/100 is the percentage of minimum number of available bytes the file system must maintain
+            warning_available = (100 - warning_threshold) * (info.capacity / 100);
+         }
 
          // Add to the list
          filesystems.emplace_back(statbuf.st_dev, shutdown_available, path_name, warning_available);
-         
-         ilog("${path_name}'s file system monitored. shutdown_available: ${shutdown_available}, capacity: ${capacity}, threshold: ${threshold}", ("path_name", path_name.string()) ("shutdown_available", shutdown_available) ("capacity", info.capacity) ("threshold", shutdown_threshold) );
+
+         uint64_t threshold = shutdown_absolute > 0 ? shutdown_absolute : shutdown_threshold;
+         ilog("${path_name}'s file system monitored. shutdown_available: ${shutdown_available}, capacity: ${capacity}, threshold: ${threshold}",
+              ("path_name", path_name.string()) ("shutdown_available", shutdown_available) ("capacity", info.capacity) ("threshold", threshold) );
```

Comment:
I think it would be nice to provide the unit here as we do with `desc` above. Maybe we should have a function that returns `{ threshold, desc }`  as this is used in multiple locations.